src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -47,6 +47,15 @@ object TestUtils  {
       .withClaim("user_id", userId)
       .build)
   )
+  def validBackendChecksToken(): OAuth2BearerToken = OAuth2BearerToken(tdrMock.getAccessToken(
```

Comment:
Nitpick: this helper refers to "valid backend checks" but it's specifically for antivirus. I'd be inclined to pass the role as a parameter to make the test setup more explicit.